### PR TITLE
Add debian repo for dpkg

### PR DESCRIPTION
**Pull request by rgayon**

### CODE REVIEW

File: debian/rules [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#! /usr/bin/make -f
+
+#export DH_VERBOSE = 1
+export PYBUILD_NAME = docker-explorer
+
+%:
+	dh $@ --with python2,python3 --buildsystem=pybuild
```
Comment (joachimmetz) [inline debian/rules]: it looks like you're only using python2 so maybe remove `,python3` ?

Comment (rgayon) [reply debian/rules]: done

**Comment by rgayon** [comment]
Updated

File: docker_explorer/de.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/python
```
Comment (Onager) [inline docker_explorer/de.py]: Perhaps /usr/bin/env python so that this works in virtualenvs as well?

File: debian/control [hunk 3]
```diff
@@ -0,0 +1,13 @@
+Source: docker-explorer
+Section: python
+Priority: extra
+Maintainer: docker-explorer Developers <docker-explorer-devs@google.com>
+Build-Depends: debhelper (>= 9), python, dh-python, python-setuptools
+Standards-Version: 3.9.5
+
+Package: docker-explorer
+Architecture: any
+Pre-Depends: dpkg (>= 1.16.1), python2.7-minimal, ${misc:Pre-Depends}
```
Comment (joachimmetz) [inline debian/control]: I don't think `Pre-Depends` is needed

File: docker_explorer/de.py (hunk 2 above)
Comment (joachimmetz) [reply docker_explorer/de.py]: I opt to make this part of the python3 changes, seeing that now only `#!/usr/bin/python2` really works

Comment (rgayon) [reply docker_explorer/de.py]: Updated #7

**Pull request merged**